yang/frr-zebra.yang

```diff
@@ -617,7 +618,6 @@ module frr-zebra {
 
         list route {
           key "prefix";
-          config false;
```

Comment:
> Zebra routing tables are not controlled by the user and can not be created/deleted manually [...]

I'm not sure about that...

@chiragshah6 any thoughts on this?

Reply:
They are created/deleted automatically when a VRF is created/deleted.

Reply:
I agree with @idryzhov there is no situation where a user can just auto-create a table via a cli command.  There is implied
say via a `ip route ... table YY` command or through a routemap in bgp via `set table ZZ`.   In these situations zebra is told about what table to install the route in via ZAPI and zebra auto-creates the specified table.  I cannot think of a situation where a end user explicitly calls out a table to be created in zebra.

Reply:
@idryzhov @donaldsharp makes sense to me now. Thanks for the clarification.